ee/vellum_ee/workflows/display/types.py

```diff
@@ -22,6 +22,12 @@
 
 WorkflowDisplayType = TypeVar("WorkflowDisplayType", bound="BaseWorkflowDisplay")
 
+NodeDisplays = Dict[Type[BaseNode], BaseNodeDisplay]
+NodeOutputDisplays = Dict[OutputReference, Tuple[Type[BaseNode], NodeOutputDisplay]]
+WorkflowOutputDisplays = Dict[BaseDescriptor, WorkflowOutputDisplay]
+EdgeDisplays = Dict[Tuple[Port, Type[BaseNode]], EdgeDisplay]
+PortDisplays = Dict[Port, PortDisplay]
```

Comment:
We're going to be editing some of these types. Since they are also redefined elsewhere, we centralize their spec